docs/dev-guides/telemetry-messages-format.md

```diff
@@ -8,141 +8,91 @@ Telemetry messages used by the IIoT Platform
 
 ## From IoT Hub - Generated by Edge Publisher
 
-### Samples Mode
-
-JSON Encoding
+### Samples Mode, JSON encoding
 
 ```json
 {
   "body": {
-    "NodeId": "http://test.org/UA/Data/#i=10845",
-    "EndpointUrl": "opc.tcp://172.17.126.1:51210/UA/Server",
-    "ApplicationUri": "urn:IndustrialHost01:OpcUaServer",
-    "DisplayName": "SByteValue",
-    "Timestamp": "2020-03-25T00:30:59.8229777Z",
-    "Status": "Good",
+    "NodeId": "nsu=http://microsoft.com/Opc/OpcPlc/;s=StepUp",
+    "EndpointUrl": "opc.tcp://opcplc:50000/",
+    "ApplicationUri": "urn:OpcPlc:opcplc",
+    "DisplayName": "StepUp",
+    "Timestamp": "2022-03-18T12:52:42.137703Z",
     "Value": {
-      "Value": -30,
-      "SourceTimestamp": "2020-03-25T00:30:59.0134194Z",
-      "ServerTimestamp": "2020-03-25T00:30:59.0134194Z"
+      "Value": 21713,
+      "SourceTimestamp": "2022-03-18T12:52:42.1327544Z",
+      "ServerTimestamp": "2022-03-18T12:52:42.1327633Z"
     },
+    "SequenceNumber": 120,
     "ExtensionFields": {
-      "EndpointId": "uat46f9f8f82fd5c1b42a7de31b5dc2c11ef418a62f",
-      "PublisherId": "uat46f9f8f82fd5c1b42a7de31b5dc2c11ef418a62f",
-      "DataSetWriterId": "uat46f9f8f82fd5c1b42a7de31b5dc2c11ef418a62f"
+      "PublisherId": "opc.tcp://opcplc:50000_D09D61EF",
+      "DataSetWriterId": "1000"
     }
   },
-  "enqueuedTime": "2020-03-25T00:31:00.078Z",
+  "enqueuedTime": "Fri Mar 18 2022 13:52:42 GMT+0100 (Central European Standard Time)",
```

Comment:
Is this really how `enqueuedTime` is reported ? Should we fix it to have the previous format which is in line with the rest of timestamps that we have ?

Reply:
So this is not a change in the formatting that we did, right ?